platform/integrations/figma_plugin/usage.mdx

```diff
@@ -19,6 +19,10 @@ To push all texts into Tolgee, first unselect all and then click **Push all**. I
 Then you will see an overview of the keys that will be pushed into Tolgee.
 By default, the plugin will create (or update) screenshots to the keys – you can disable this by unchecking the checkbox. To confirm the settings, click **Push to Tolgee**.
 
+:::info
+The plugin uses figma `frames` to decide what is a "screen" so if the screenshots are not properly created check if you wrap each screen in `frame` (if a translation is in multiple frames the one top-level is taken).
```

Comment:
"If a translation is in multiple frames the one top-level is taken" - This looks too important to be in brackets.
"translation" -> I would use "string" where possible. Translation refers to the "process" or possibly specifically the "target value". 

```suggestion
The plugin uses figma `frames` to decide what is a "screen" so if the screenshots are not properly created check if you wrap each screen in `frame`. If a translation is in multiple frames the one top-level is taken.
```